### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by RomanEmmons**
Implements New Relic, Artillery and some seeding touch ups.

### CODE REVIEW

File: server/routes/rsvp.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    `select * from members where attending=${req.params.eventId}`,
+    // `select * from members where name="Tressie Gleason II";`,
+    (err, results) => {
+      if (err) {
+        console.log('err!', err);
+        res.status(404).json(`Error: ${err}`);
+      } else {
+        console.log('results', results);
+        var attendees = [];
+        _.forEach(results, user => {
```
Comment (bellcd) [inline server/routes/rsvp.js]: There's probably a little room here for extracting the _.forEach() / redis .set() calls into a separate module (maybe another middleware?), if you wanted to.

File: server/routes/rsvp.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+          const person = {
+            name: user.name,
+            avatar: user.avatar,
+            thumbnail: user.thumbnail
+          };
+          attendees.push(person);
+        });
+        client.set(
+          req.params.eventId,
+          JSON.stringify(attendees),
```
Comment (bellcd) [inline server/routes/rsvp.js]: any particular reason you're [storing as JSON in redis](https://redislabs.com/redis-best-practices/data-storage-patterns/json-storage/)? perhaps consider [hash storage](https://redislabs.com/redis-best-practices/data-storage-patterns/object-hash-storage/)?

File: server/routes/rsvp.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+          }
+        );
+      }
+    }
+  );
+
+  // put that updates eventLimit and setLimit
+  router.route('/:eventId/:eventLimit/:setLimit').put((req, res) => {
+    db.query(
+      `update events set eventLimit=${req.params.eventLimit}, setLimit=${req.params.setLimit} where id=${req.params.eventId}`,
```
Comment (bellcd) [inline server/routes/rsvp.js]: [escaping user provided data before using in a SQL query?](https://www.npmjs.com/package/mysql#escaping-query-values)

File: server/index.js [hunk 4]
```diff
@@ -1,11 +1,36 @@
...
 
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
 app.use(express.static('public'));
 
+const cache = (req, res, next) => {
+  client.get(toString(req.params.eventId), (err, data) => {
+    if (err) {
+      throw err;
```
[the remark this answers is not in the document]
Comment (RomanEmmons) [reply server/index.js]: Good eye!

**Pull request merged**